.gitignore

```diff
@@ -1,3 +1,5 @@
+.DS_Store
```

Comment:
Can you please check and see why we have this line again when a previous PR had already added this line to master?

Reply:
fixed